spec/system/courses/report_spec.rb

```diff
@@ -405,15 +411,21 @@
              target_group: target_group_l0
     end
 
-    scenario "checks status of total targets completed in report" do
+    scenario "checks status of total targets completed and targets read in report" do
       sign_in_user student.user, referrer: report_course_path(course)
 
-      # Check that level zero targets are not counted in the targets overview
-      within("div[aria-label='target-completion-status']") do
-        expect(page).to have_content("66%")
+      # Check that level zero assignments are counted in the targets overview
```

Comment:
This scenario's parent context was checking whether the special behaviour for level zero targets was being applied - but that special behaviour isn't there any more, so I don't think the context is required.